org.eclipse.xtext.tests/src-gen/org/eclipse/xtext/serializer/services/SyntacticSequencerTestLanguageGrammarAccess.java

```diff
@@ -1580,6 +1618,22 @@ public ParserRule getFragment1Rule() {
 		return getFragment1Access().getRule();
 	}
 	
+	//Bug398890:
+	//	C_COMMENT_END name=ID;
+	public Bug398890Elements getBug398890Access() {
+		return pBug398890;
+	}
+	
+	public ParserRule getBug398890Rule() {
+		return getBug398890Access().getRule();
+	}
+	
+	//terminal C_COMMENT_END:
+	//	'* /';
```

Comment:
The space between the * and the / looks like a bug, like an attempt to prevent exactly the kind of bug you're fixing, even though it's not necessary since here single-line-comments are used.